packages/asc-ui/src/components/TextField/TextField.tsx

```diff
@@ -15,6 +16,7 @@ export interface TextFieldProps extends InputProps {
   labelStyle?: object
   errorStyle?: object
   onClear?: () => void
+  autoFocus?: boolean
```

Comment:
Can you add this functionality to a story of `TextField.stories.tsx` as well?

Reply:
Done